pyro/infer/util.py

```diff
@@ -267,3 +267,11 @@ def compute_expectation(self, costs):
 
         LAST_CACHE_SIZE[0] = count_cached_ops(cache)
         return expected_cost
+
+
+def _check_fully_reparametrized(guide_site):
```

Comment:
If this is used outside of the file in which it defined, then its name should not begin with underscore.

Reply:
thanks, done